code/helpers/vector.dm

```diff
@@ -77,6 +82,14 @@ Inputs:
 		coord_x += inc_x
 		coord_y += inc_y
 
+		// The spinning x strikes the y in the z!
+		if(spin_counter != -1)
+			spin_counter++
+			if(spin_counter >= 8) spin_counter = 0
+			var/matrix/M = matrix()
+			M.Turn(90 * round(spin_counter/2))
```

Comment:
I would like to avoid BYOND's anti aliasing if possible, can you spin things at preset angles via animate?